src/server_manager/model/digitalocean.ts

```diff
@@ -38,10 +38,19 @@ export interface RegionOption extends location.CloudLocationOption {
   readonly cloudLocation: Region;
 }
 
-export enum Status {
-  ACTIVE,
-  EMAIL_UNVERIFIED,
-  MISSING_BILLING_INFORMATION,
+export interface Status {
+  // The account has not had any billing info added yet.
+  readonly needsBillingInfo: boolean;
+  // The account has not had an email address added yet.
+  readonly needsEmailVerification: boolean;
+  // The maximum number of droplets this account can create.
+  readonly dropletLimit: number;
+  // The account cannot add any more droplets.
+  readonly hasReachedLimit: boolean;
+  // A warning message from DigitalOcean.
+  readonly warning?: string;
+  // A string with detailed status info
+  readonly detail?: string;
```

Comment:
We should either keep the status here and format the string in the app code, or add it to the warning string (and delete the detail field).

Reply:
Added to the warning string.